keps/sig-api-machinery/1929-built-in-default/README.md

```diff
@@ -400,10 +402,12 @@ which would have the following behavior, both for CRD types and built-in types:
 
 #### Non-pointer fields
 
-Non-pointer fields are also important, since they have a specific unmarshalling mechanism in Go.
+Non-pointer fields are also important, since they have a specific
+unmarshalling mechanism in Go.
 
-For CRDs, we can assume that invalid zero-values are omitted, otherwise
-the objects would technically be invalid.
+For CRDs, invalid zero-values are omitted, otherwise the objects would
+technically be invalid. We can safely default these field if they are
```

Comment:
I am still lost in these paragraphs. What do you want to say? What is motivation, what is a proposed change in behaviour?